src/content-handlers/iiif/modules/uv-pdfheaderpanel-module/PDFHeaderPanel.ts

```diff
@@ -160,18 +160,26 @@ export class PDFHeaderPanel extends HeaderPanel<
 
     if (this._pageIndex === 1) {
       this.$firstButton.disable();
+      this.$firstButton.attr('tabindex', '-1');

```

Comment:
In #1093, @Saira-A used `.attr("disabled", "disabled")` and `.removeAttr("disabled")` instead of directly manipulating the `tabindex`. I think that might be a slightly broader solution to the same problem. Did you try that here? If not, do you think it's worth revising this code to match for consistency?

Reply:
Updated and it seems to work for me!